packages/font-icons/scss/_variables.scss

```diff
@@ -1,5 +1,6 @@
 $ki-font-family: "WebComponentsIcons" !default;
 $ki-icon-size: 16px !default;
+$ki-icon-padding: 4px !default;
```

Comment:
in order to test this currently in the default, bootstrap and material theme, we'll need to add it to default/scss/icon/ as well

Reply:
I will close this PR and open a new one in the kendo-themes repo in that case.